.github/workflows/test.yml

```diff
@@ -175,15 +175,6 @@ jobs:
     - run: npm ls --all || true
     - name: npm test
       run: npm test
-    - uses: inception-health/otel-upload-test-artifact-action@v1
```

Comment:
Note for reviewer: removing this will make disappear the artefacts attached to the test actions. IIUC this transforms the tests results to an OTEL trace with data in JSON format for inspection but I don't recall using it.

Reply:
This was added in #2805 as *part* of a change to support OTel-based tracing of our CI workflows. That tracing of CI workflows was dropped in #4024, but that PR forgot to remove this part.